netbox/utilities/forms/mixins.py

```diff
@@ -44,3 +46,33 @@ def clean(self):
                 "This object has been modified since the form was rendered. Please consult the object's change "
                 "log for details."
             ))
+
+
+class DistanceValidationMixin(forms.Form):
```

Comment:
Can we instead just enforce a static maximum value of 100,000 (or similar) on `distance`? So long as `_abs_distance` is adjusted to support the maximum possible _absolute_ value, this should be sufficient.

For example, assume a maximum value of 100,000. The longest available unit is mile; 100,000 miles equals 160,934,000 meters. We just need to ensure that `_abs_distance` can store this theoretical maximum value.

Reply:
Fair enough.

The thinking behind making such a large change in potential distance upper limit was that, since there appears to be a negligible difference to storage requirements, why not set some ridiculous upper bound in absolute distance that could represent any potential circuit or wireless link possibly constructed. I suppose that qualifies as a premature optimization.